client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue

```diff
@@ -162,6 +178,21 @@ export default {
         .then(() => this.$emit('deleted'))
     },
 
+    async saveSignLanguage() {
+      const payload = {
+        id: this.currentCustomerId,
+        type: 'Customer',
+        attributes: {
+          ...this.customerList[this.currentCustomerId].attributes,
+
+        }
```

Comment:
```suggestion
        attributes: { ...this.customerList[this.currentCustomerId].attributes }
```

please remove the blank line and the comma (optional: you could also write it in one line)

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2212/commits/7b6ce1f81656a91c4012b3bcff690b124e511104

Reply:
I'd remove that comma too but I think its fine